scripts/ad4m-connect-auth.sh

```
@@ -0,0 +1,280 @@
+
+    CHROME_FLAGS=(
+        --remote-debugging-port="$CDP_PORT"
+        --user-data-dir="$PROFILE"
+        --no-first-run
+        --no-default-browser-check
+    )
+    $HEADLESS && CHROME_FLAGS+=(--headless=new)
+
+    "$CHROME_BIN" "${CHROME_FLAGS[@]}" "$FLUX_URL" &>/dev/null &
+    CHROME_PID=$!
+    trap "kill $CHROME_PID 2>/dev/null" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)